Inspicio/ClaimsPrincipalExtensions/ProfileNameClaimsPrincipalFactory.cs

```diff
@@ -0,0 +1,47 @@
+﻿
+    // Jack Lloyd [06/07/17]
+/*
+    Overriding the UserClaimsprincipalFactory
+    in order to add a 'ProfileName' claim 
+    into Claims of Authenticated Users
+ */
+
+using Inspicio.Models;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using System.Security.Claims;
+
+using System.Threading.Tasks;
+
+namespace Inspicio.ClaimsPrincipalExtensions
+{
+    public class ProfileNameClaimsPrincipalFactory : UserClaimsPrincipalFactory<ApplicationUser, IdentityRole>
+    {
+        public ProfileNameClaimsPrincipalFactory (
+            UserManager<ApplicationUser> userManager,
+            RoleManager<IdentityRole> roleManager,
+            IOptions<IdentityOptions> optionsAccessor ) : base( userManager, roleManager, optionsAccessor )
+        {
+
+        }
+
+        public async override Task<ClaimsPrincipal> CreateAsync( ApplicationUser user )
+        {
+
+                // Jack Lloyd [06/07/17]
```

Comment:
Including authors in comments gets a bit confusing when somebody else comes along and makes a change to that part of the code. `bit blame` is the reliable way to work out who wrote something (you can get the `blame` info by right clicking in a source file in VS and choosing "Source control" form the context menu).